### PR TITLE
Disable jails tab for unified states

### LINKED ISSUE #120: Disable jails tab for "unified system" states.
A limited set of states are "unified states," which combine prison and jails in the same system. Because of this, it could be misleading to allow users to click into the Jails tab (especially since there is often no data available there for those states). 

The following states are affected by this: 
- Alaska (AK)
- Connecticut (CT)
- Delaware (DE) 
- Hawaii (HI)
- Vermont (VT) 

The following would be the ideal behavior: 
- P0: Disable the Jails tab for those states. Clicking the Jails tab should do nothing; hovering should not change the cursor style, and the 
    - P1: Ideally, the label should then read "Jails (Not Applicable)" to make it clear that this is not an available option.
- P0: Add text to the header/introductory text that reads as follows. This should be enabled by #113 .
> This state operates a "unified corrections system," which combines the jail and prison systems. As such, some of the numbers below may include pre-trial populations.

### PR DESCRIPTION
**Pull request by colincadams**
## Description of the change

This disables the jails tab for states which have unified systems. In the future we may break out pre-trial populations to the jails tab for these states, but it is a large change so for now we just disable it.

## Type of change

- [x] New feature (non-breaking change that adds functionality)

## Related issues

Closes #120 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/components/MainPage/Tabs.js [hunk 1]
```diff
@@ -36,18 +36,23 @@ const Tabs = ({ activeTab, onTabChange }) => {
...
+        })}
+        onClick={isUnified ? null : createOnTabChange(JAILS)}
       >
-        Jails
+        Jails{isUnified && " (Not Applicable)"}
       </button>
     </div>
   );
 };
+// This state operates a "unified corrections system," which combines the jail and prison systems. As such, some of the numbers below may include pre-trial populations.
```
Comment (macfarlandian) [inline src/components/MainPage/Tabs.js:50]: this comment seems out of place? or maybe just needs to be deleted

File: src/utils/getIsUnified.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    case "US_CT":
+      return true;
+    case "US_DE":
+      return true;
+    case "US_HI":
+      return true;
+    case "US_RI":
+      return true;
+    case "US_VT":
+      return true;
```
Comment (macfarlandian) [inline src/utils/getIsUnified.js:30]: I think you could have all these cases fall through to a single `return true` statement? might be a little easier to read

**Review by macfarlandian — APPROVED** [review]

**Review by jessex — APPROVED** [review]
No new comments from me beyond what Ian shared!

File: src/components/MainPage/Tabs.js (hunk 1 above)
Comment (colincadams) [reply src/components/MainPage/Tabs.js]: yeah this was a bad paste, deleted!

**Pull request merged**